core/app/models/spree/taxon.rb

```diff
@@ -95,6 +98,23 @@ def child_index=(idx)
       move_to_child_with_index(parent, idx.to_i) unless new_record?
     end
 
+    def move_to_child_with_index(node, index)
+      # closure_tree does not provide any method that move nodes regarding position
+      # this method overrides move_to_child_with_index offered by awesome_nested_set
```

Comment:
Wait, "overrides"? Is `awesome_nested_set` still around to be overridden, or does the method just follow the signature from `awesome_nested_set`?

Also: We tend to have comments above method definitions, along with a little bit of [YARD doc](https://gist.github.com/chetan/1827484). If you could move that once we've come to a decision about this PR, that'd be great.